assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -80,6 +81,7 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi }) => {
         context: 'edit',
         _web_stories_envelope: true,
         search: searchTerm || undefined,
+        author: author || undefined,
```

Comment:
I believe this file is part of the Story editor, if so we need to check the test cases are working fine for that.

Reply:
@divyarajmasani In the automated check list, both e2e and js unit tests are passed. So I guess it's OK.